src/adminPages/modal/AddUserForm.js

```diff
@@ -0,0 +1,280 @@
+                                    <label
+                                        className='generic-input-label cost-tracker-input-label'
+                                        htmlFor='active-state'
+                                    >
+                                        Permissions
+                                    </label>
+
+                                    <Controller
+                                        as={iconTypeSelector}
+                                        name='iconType'
+                                        control={control}
+                                        defaultValue=''
+                                        rules={{
+                                            required: true,
+                                        }}
+                                        help={errors.iconType && 'Please select a value'}
+                                    />
+                                    <p className='input-error-message'>
+                                        {errors.iconType && 'Please select a value'}
+                                    </p>
+                                </div>
+                            </Col>
+                            <Col md={8}>
+                                <div className='cost-tracker-input-container'>
+                                    <label
+                                        className='generic-input-label cost-tracker-input-label'
+                                        htmlFor='equipment-purchase-date'
+                                    >
+                                        Date
+                                    </label>
+                                    <Controller
+                                        as={dateAddedPicker}
+                                        name='dateAdded'
+                                        control={control}
+                                        defaultValue=''
+                                        rules={{
+                                            required: true,
+                                        }}
+                                        validateStatus={
+                                            errors.dateAdded && 'Please enter a date' ? 'error' : ''
+                                        }
+                                        help={errors.dateAdded && 'Please enter a date'}
+                                    />
+                                    <p className='input-error-message'>
+                                        {errors.dateAdded && 'Please enter a date'}
+                                    </p>
+                                </div>
+                            </Col>
+                            <Col md={8}>
+                                <button className='generic-submit-button cost-tracker-form-submit-button'>
+                                    Add
+                                </button>
+                            </Col>
```

Comment:
Nice work @Ebenezy , please lets refactor this form to use antd design forms.
using ref makes it hard to manage.
With Antd design forms, validations and submitions would be easy.
Please use antd design forms on all pages that involve forms